README.md

```diff
@@ -1,2 +1,16 @@
-# hw01
-Repository for first CFSS homework assignment
+---
+title: "A Brief Introdution"
+author: "Eleanor Koh"
+date: "11/01/2017"
+output: html_document
```

Comment:
For a plain Markdown (`.md`) document this header is generally not necessary. But absolutely useful once we get to R Markdown (`.Rmd`) files